src/data/ACTIONS/root/AST.ts

```diff
@@ -409,60 +409,64 @@ export const AST = ensureActions({
 	// Arcana cast
 	// -----
 
+	//PLAY_I
 	THE_BALANCE: {
-		id: 4401,
+		id: 37023,
 		name: 'The Balance',
 		icon: iconUrl(3110),
 		cooldown: 1000,
 		statusesApplied: ['THE_BALANCE'],
 		cooldownGroup: 2,
 	},
 
-	THE_BOLE: {
-		id: 4404,
-		name: 'The Bole',
-		icon: iconUrl(3111),
+	THE_SPEAR: {
+		id: 37026,
+		name: 'The Spear',
+		icon: iconUrl(3113),
 		cooldown: 1000,
-		statusesApplied: ['THE_BOLE'],
+		statusesApplied: ['THE_SPEAR'],
 		cooldownGroup: 2,
 	},
 
+	//PLAY_II
 	THE_ARROW: {
-		id: 4402,
+		id: 37024,
 		name: 'The Arrow',
 		icon: iconUrl(3112),
 		cooldown: 1000,
 		statusesApplied: ['THE_ARROW'],
-		cooldownGroup: 2,
+		cooldownGroup: 100, //not true cooldown group. synthed to be able to split in action timeline
 	},
 
-	THE_SPEAR: {
-		id: 4403,
-		name: 'The Spear',
-		icon: iconUrl(3113),
+	THE_BOLE: {
+		id: 37027,
+		name: 'The Bole',
+		icon: iconUrl(3111),
 		cooldown: 1000,
-		statusesApplied: ['THE_SPEAR'],
-		cooldownGroup: 2,
+		statusesApplied: ['THE_BOLE'],
+		cooldownGroup: 100, //not true cooldown group. synthed to be able to split in action timeline
 	},
 
+	//PLAY_III
 	THE_EWER: {
-		id: 4405,
+		id: 37028,
 		name: 'The Ewer',
 		icon: iconUrl(3114),
 		cooldown: 1000,
 		statusesApplied: ['THE_EWER'],
-		cooldownGroup: 2,
+		cooldownGroup: 101, //not true cooldown group. synthed to be able to split in action timeline
 	},
 
 	THE_SPIRE: {
-		id: 4406,
+		id: 37025,
 		name: 'The Spire',
 		icon: iconUrl(3115),
 		cooldown: 1000,
 		statusesApplied: ['THE_SPIRE'],
-		cooldownGroup: 2,
+		cooldownGroup: 101, //not true cooldown group. synthed to be able to split in action timeline
```

Comment:
(As stated in earlier comment) My intention was to group by defensive and offensive. If you agree, please combine these cooldown groups.

Reply:
updated!
here is the new timeline
![image](https://github.com/xivanalysis/xivanalysis/assets/89435408/c15a41fa-70c6-4351-9793-40255608bf4c)